src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  display: flex;
+  width: 200px;
+  background-color: #fff;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition-duration: 300ms;
+
+  &__image {
+    display: flex;
+    height: 134px;
+    width: 160px;
+    margin: 32px auto 0;
+  }
+
+  &__product-name {
+    display: flex;
+    font-weight: 500;
+    padding-inline: $padding;
```

Comment:
The variable `$padding` is used here, but it's not defined within this file. Ensure that it's defined in a parent file or imported correctly.